framework/composer.json

```diff
@@ -65,7 +65,7 @@
         "ezyang/htmlpurifier": "~4.6",
         "cebe/markdown": "~1.0.0 | ~1.1.0",
         "bower-asset/jquery": "2.2.*@stable | 2.1.*@stable | 1.11.*@stable | 1.12.*@stable",
-        "bower-asset/jquery.inputmask": "~3.2.2",
+        "bower-asset/jquery.inputmask": "~3.3.3",
```

Comment:
should be `~3.3.3 | ~3.2.2` instead to allow upgrading yii without the need to upgrade the inputmask lib.